### DISCUSSION TITLE
small idea

### DISCUSSION THREAD

**Discussion by sznlner** [ppy/osu#21745] [Ideas]
just make it so when we're on mobile and go to the friend tab it shows when were they seen online because it doesn't show anything until i change it to desktop mode (it's not a big of a deal but it's better than nothing)

**Comment by ghost** [comment]
is this for the website?

**Reply by sznlner** [reply]
yes

**Reply by Rekunan** [reply]
You should open an issue in the [osu-web](https://github.com/ppy/osu-web) repo then, this repo is for Lazer.
